.github/workflows/build.yml

```
@@ -90,6 +90,40 @@ jobs:
       with:
         name: debs
         path: qdigidoc4*.*
+  fedora:
+    name: Build on Fedora to ${{ matrix.container }}
+    runs-on: ubuntu-latest
+    container: ${{ matrix.container }}
+    strategy:
+      matrix:
+        container: ['fedora:33']
+    env:
+      MAKEFLAGS: -j3
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+      with:
+        submodules: recursive
+    - name: Install Build Deps
+      run: dnf groupinstall -y "Development Tools" "Development Libraries" fedora-packager
+    - name: Install Deps
+      run: |
+        dnf install -y \
+          cmake openssl-devel xerces-c-devel \
+          xml-security-c-devel zlib-devel vim-common \
+          https://www.codesynthesis.com/download/xsd/4.0/linux-gnu/x86_64/xsd-4.0.0-1.x86_64.rpm
+    - name: Build
+      run: |
+        mkdir build
+        cd build
+        cmake ..
```

```suggestion
        cmake -DCMAKE_INSTALL_SYSCONFDIR=/etc ..
```
@metsma Should I do this here too?

---

I don't think digidoc installs anything under etc

---

```suggestion
        cmake -DCMAKE_INSTALL_PREFIX=/usr ..
```
Can you add for now to see if it fixes
CMake Error: failed to create symbolic link '/__w/DigiDoc4-Client/DigiDoc4-Client/build/_CPack_Packages/Linux/RPM/qdigidoc4-4.2.11.53-1.x86_64//usr/local/share/icons/hicolor/16x16/mimetypes/application-vnd.etsi.asic-s+zip.png': no such file or directory